lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -199,6 +199,8 @@ struct BindFileInfo {
 
 class LowerLayersPass
     : public circt::firrtl::impl::LowerLayersBase<LowerLayersPass> {
+  using LowerLayersBase::LowerLayersBase;
```

Comment:
Nit: I've seen the pattern in upstream of the slightly terser:


```suggestion
  using Base::Base;
```

Reply:
TIL :+1: 

```c++
template <typename DerivedT>
class LowerLayersBase : public ::mlir::OperationPass<firrtl::CircuitOp> {
public:
  using Base = LowerLayersBase;
 ...
```